src/getTakwimuPage.js

```diff
@@ -14,3 +14,17 @@ export default async function getTakwimuPage(type) {
 
   return config;
 }
+
+export async function getSitePage(slug) {
+  console.time('download');
+  const res = await fetch(
+    `https://takwimutech.wpengine.com/wp-json/wp/v2/pages?slug=${slug}`
```

Comment:
Possible to set this URL from config so it's easier to change in future?

Reply:
e.g `${WP_BACKEND_URL}/wp-json/wp/v2/pages?slug=${slug}`

Reply:
@DavidLemayian sure!